src/GraphQL/Types/TypeInformation.cs

```diff
@@ -105,6 +107,39 @@ public TypeInformation(PropertyInfo propertyInfo, bool isInputProperty)
         {
             var typeTree = Interpret(new NullabilityInfoContext().Create(propertyInfo), isInputProperty);
 
+            ProcessTypeTree(typeTree, isInputProperty);
```

Comment:
```suggestion
            ProcessTypeTree(typeTree, isInput);
```

\+ Interpret(NullabilityInfo info, bool isInputProperty) -> Interpret(NullabilityInfo info, bool isInput)

Reply:
done (both)